netbench-cdk/lib/netbench.ts

```diff
@@ -166,4 +168,35 @@ export class NetbenchInfra extends cdk.Stack {
         return netbenchBucket;
     };
 
+    private createMonitorLambda(): any {
+        const monitorLambda = new cdk.aws_lambda.Function(this, "netbenchMonitor", {
+            runtime: cdk.aws_lambda.Runtime.PYTHON_3_12,
+            handler: "index.lambda_handler",
+            code: cdk.aws_lambda.Code.fromInline(readFileSync('./netbench-monitor/handler.py', 'utf-8')),
+            timeout: cdk.Duration.seconds(15)
+        })
+        //monitorLambda.addToRolePolicy(PolicyStatement())
+        const describePolicy = cdk.aws_iam.PolicyStatement.fromJson({
+            "Sid": "VisualEditor0",
```

Comment:
is this the name? Can we give a more descriptive one for when we look at it in the console?

Reply:
That Sid doesn't need to be there, pulling it. The policy is inline, so you'd need to find the role attached to the lambda to view it; it's not straightforward... here is an example name: `NetbenchInfraDev-dougch-netbenchMonitorServiceRoleF-dix1MPm2DT1F`